src/Executor/Executor.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Fx.CommandLine;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.Execute
+{
+    public class Executor
+    {
+        public string configFilePath;
+        public string configFileName = "config.json";
+        public string CommandSelectedByUser { get; set; }
+        
+        public Dictionary<string, List<string>> DevWorkflowCommands { get; set; }
+        public Dictionary<string, string> SettingParameters { get; set; }
+        public Dictionary<string, string> CommandParameters { get; set; }
+        
+        public Executor()
+        {
+            SettingParameters = new Dictionary<string, string>();
+            CommandParameters = new Dictionary<string, string>();
+            DevWorkflowCommands = new Dictionary<string, List<string>>();
+            string executorDirectory = Path.GetDirectoryName(typeof(Executor).GetTypeInfo().Assembly.Location);
+            configFilePath = Path.GetFullPath(Path.Combine(executorDirectory, @"..\"));
```

Comment:
Ok. So by default it would like for the file there, otherwise the file path should be the first parameter sent to the executor.
I updated the documentation to show that